wordpress-mobile/WordPress-Android
# [Home Page Picker] - Fixes title and action bar overlap

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/13558

## Description
Ellipsize centered title to prevent overlapping in some languages

## To test:

<details>
<summary><strong>To start the Home Page Picker screen</strong></summary>

1. Start the site creation flow (e.g. *Choose site > Add button*)
1. Select the *Create WordPress.com site* option
</details>

<details>
<summary><strong>To switch language</strong></summary>

1. From the My Site screen select the user avatar
1. Select App Settings (third option)
1. Tap on the language selection (first option)
</details>

1. Switch the app to language with large words (e.g. Russian or German)
1. Open the Home Page Picker screen
1. Verify that the *Skip* button is visible

## Screenshots

|English|Russian|German|
|---|---|---|
|![device-2020-12-08-173013](https://user-images.githubusercontent.com/304044/101505248-d0fe8080-397c-11eb-9a66-2806fffd38c8.png)|![device-2020-12-08-173121](https://user-images.githubusercontent.com/304044/101505283-dc51ac00-397c-11eb-88eb-f0a0dc78a7ab.png)|![device-2020-12-08-172906](https://user-images.githubusercontent.com/304044/101505299-e1166000-397c-11eb-8193-8926b6cedd1c.png)|

PR submission checklist:
